contracts/StableAsset/ERC20StableAsset.sol

```diff
@@ -10,32 +10,32 @@ import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Pausable.sol";
 import "@openzeppelin/contracts/utils/Strings.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-abstract contract ERC20StableShare is Ownable, ERC20Pausable {
+abstract contract ERC20StableAsset is Ownable, ERC20Pausable {
     uint8 private _decimals;
-    string private _isinCode;
+    string private _asset_code;
     string private _token_symbol;
```

Comment:
Symbol and name are part of base ERC20 .. we should see if we need symbol and name at this level